providers/zohocrm/errors.go

```diff
@@ -0,0 +1,45 @@
+package zohocrm
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+/*
+The normal standard response from ZohoCRM look like this:
+{
+	"data": [{...}, {...}],
```

Comment:
the key should be "user" or "profile" or something not "data"

Reply:
This is specific to Reading Records API. Updated comments to include this and a link.

Reply:
got it, thanks for the explanation and thanks for the doc link!